app/src/components/AddFeedbackForm/index.js

```diff
@@ -0,0 +1,161 @@
+import React, { Component, PropTypes } from 'react';
+import Form from 'grommet/components/form';
+import FormField from 'grommet/components/formfield';
+import FormFields from 'grommet/components/formfields';
+import Footer from 'grommet/components/footer';
+import Button from 'grommet/components/button';
+import Headline from 'grommet/components/Headline';
+import Heading from 'grommet/components/Heading';
+import Paragraph from 'grommet/components/Paragraph';
+// import LinkNextIcon from 'grommet/components/icons/base/LinkNext';
+import Menu from 'grommet/components/menu';
+import cssModules from 'react-css-modules';
+import styles from './index.module.scss';
+
+const itemInvalid = (item) =>
+  item.touched && item.error;
+
+class AddFeedbackForm extends Component {
+  constructor() {
+    super();
+    this.formIsInvalid = this.formIsInvalid.bind(this);
+  }
+  formIsInvalid() {
+    const {
+      nameInput,
+      descriptionInput,
+      urlInput,
+    } = this.props;
+    return itemInvalid(nameInput) ||
+      itemInvalid(descriptionInput) ||
+      itemInvalid(urlInput);
+  }
+  render() {
+    const {
+      nameInput,
+      urlInput,
+      descriptionInput,
+      onSubmitFeedback,
+      onClear,
+      user,
+      message,
+    } = this.props;
+    return (
+      <div>
+        {message &&
+          <Heading
+            tag="h4"
+            align="center"
+          >
+            <Paragraph>
+              {message}
+            </Paragraph>
+          </Heading>
+        }
+      <Headline
```

Comment:
Small stylistic suggestion here.  You want to always nest children below their parent, just like with HTML.

The `<Headline` and the rest of your JSX starts on the same line as the surrounding Div. See if you can get all of that indented so that it starts on the same line as `{message`